PINCache/PINApplicationBackgroundTask.m

```diff
@@ -0,0 +1,61 @@
+//
+//  PINApplicationBackgroundTask.m
+//  PINCache
+//
+//  Created by Ben Asher on 2/11/16.
+//  Copyright © 2016 Pinterest. All rights reserved.
+//
+
+#if !TARGET_OS_WATCH
+#if __IPHONE_OS_VERSION_MIN_REQUIRED >= __IPHONE_4_0
+#import <UIKit/UIKit.h>
```

Comment:
Oh this looks like the 2 are related, but actually a lot of things in UIApplication.h aren't available, so I just wrapped the whole class in this.